library/src/scripts/content/UserContent.story.tsx

```diff
@@ -46,7 +46,7 @@ storiesOf("CoreComponents/UserContent", module)
            }
        }
        </pre><p><br></p><h2>H2 Here. Spoiler next</h2><div class="spoiler"><div contenteditable="false" class="spoiler-buttonContainer">
-       <button class="iconButton button-spoiler js-toggleSpoiler">
+       <button title="Toggle Spoiler" class="iconButton button-spoiler js-toggleSpoiler">
```

Comment:
Should this also be wrapped in the translation function?

Reply:
I checked with Adam, it was just a quick dump of rendered HTML. It will eventually be replaced by "real" rendered content.